constants/index.ts

```diff
@@ -76,3 +76,61 @@ export const DISCORD_URL =
   process.env.NEXT_PUBLIC_DISCORD_URL || 'https://discord.com/invite/QFkNsjTkzD';
 export const GITHUB_URL =
   process.env.NEXT_PUBLIC_GITHUB_URL || 'https://github.com/SwaprHQ/presagio';
+
+const formatEventName = (eventName: string) =>
+  eventName.split(' ').join('-').toLocaleLowerCase();
+export const FA_EVENTS = {
```

Comment:
Probably it's better to have a analytics file, right?

Reply:
Yes! I agree, this `FA_EVENTS` object feels out of place now hahaha, I'll move it, maybe to its own module (`@/analytics`) or inside the `useAnalytics` hook file